release/release.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -ex
+
+# shellcheck source=/dev/null
+source "$KOKORO_GFILE_DIR/common.sh"
+"$KOKORO_GFILE_DIR/verify-commits.sh"
+cd github/debian-docker
```

Comment:
Do you need to cd here? Could you call it in that path instead?

Reply:
Nope. Fixed.